src/apiController.js

```diff
@@ -97,87 +97,74 @@ let apiController = {
       console.error('Oh noes ', err);
     });
   },
+
+
   //Questions and Answers API calls
-  getQuestions: (productId, { pages = 1, count = 5 } = {}) => {
-    let params = { page: pages, count: count, product_id: productId };
-    return axios
-      .get('/qa/questions', { params })
+  //according to the docs the server expects one object with product_id, page, count inside as a parameter
+  getQuestions: ({ productId = 21111, page = 1, count = 5 }) => {
```

Comment:
I'm not sure you necessarily want to have a default value for productId.

Also, this function now operates differently than the rest of them (even in your component), where the ID is in the object, rather than being a seperate parameter. I would make sure it's consistent one way or the other.

Reply:
the code was working but I just realized I was trying to fix a bug in the wrong place. Will try to maintain consistent styling instead